examples/main/README.md

```diff
@@ -17,60 +17,68 @@ This example program allows you to use various LLaMA language models in an easy
 
 To get started right away, run the following command, making sure to use the correct path for the model you have:
 
-#### Unix-based systems (Linux, macOS, etc.):
+### Unix-based systems (Linux, macOS, etc.):
+
+##### Input prompt (One-and-done)
 
 ```bash
-./llama-cli -m models/7B/ggml-model.bin --prompt "Once upon a time"
+./llama-cli -m models/gemma-1.1-7b-it.Q4_K_M.gguf --prompt "Once upon a time"
 ```
+##### Conversation mode (Allow for continuous interaction with the model)
 
-#### Windows:
-
-```powershell
-llama-cli.exe -m models\7B\ggml-model.bin --prompt "Once upon a time"
+```bash
+./llama-cli -m models/gemma-1.1-7b-it.Q4_K_M.gguf -cnv --ctx_size 0 --chat-template llama3
```

Comment:
specifying `--chat-template` here is misleading, why would user want to use gemma model with llama3 template?

`--ctx_size` is also not necessary

Reply:
ups, I did everything for llama and then realised that we do not have any llama models in ggml repo and replaced everything gemma.